Error in LOG definitions of code generator

**Platform:** Windows

Hi,

There is an error in code generator, the LOG definitions (`LOGE_CODE_GENERATOR`, `LOGI_CODE_GENERATOR`) seem to be undeclared.

The error surfaces particularly when building a model. 
See:
```
F:\genn-master\src\genn\generator\generator.cc(79,17): error C2065: 'LOGE_CODE_GENERATOR': undeclared iden
tifier [F:\genn-master\src\genn\generator\generator.vcxproj]
F:\genn-master\src\genn\generator\generator.cc(94,13): error C2065: 'LOGI_CODE_GENERATOR': undeclared iden
tifier [F:\genn-master\src\genn\generator\generator.vcxproj]
F:\genn-master\src\genn\generator\generator.cc(105,13): error C2065: 'LOGI_CODE_GENERATOR': undeclared ide
ntifier [F:\genn-master\src\genn\generator\generator.vcxproj]
```

--- a/src/genn/generator/generator.cc
+++ b/src/genn/generator/generator.cc
@@ -76,7 +76,7 @@ int main(int argc,     //!< number of arguments; expected to be 2
             // Create a new GUID for project
             GUID guid;
             if(::CoCreateGuid(&guid) != S_OK) {
-                LOGE_CODE_GENERATOR << "Unable to generate project GUID";
+                LOGE_CODE_GEN << "Unable to generate project GUID";
                 return EXIT_FAILURE;
             }
 
@@ -91,7 +91,7 @@ int main(int argc,     //!< number of arguments; expected to be 2
 
             // Use result as project GUID string
             projectGUIDString = projectGUIDStream.str();
-            LOGI_CODE_GENERATOR << "Generated new project GUID:" << projectGUIDString;
+            LOGI_CODE_GEN << "Generated new project GUID:" << projectGUIDString;
 
             // Write GUID to project GUID file
             std::ofstream projectGUIDFile(projectGUIDFilename.str());
@@ -102,7 +102,7 @@ int main(int argc,     //!< number of arguments; expected to be 2
             // Read GUID from project GUID file
             std::ifstream projectGUIDFile(projectGUIDFilename.str());
             std::getline(projectGUIDFile, projectGUIDString);
-            LOGI_CODE_GENERATOR << "Using previously generated project GUID:" << projectGUIDString;
+            LOGI_CODE_GEN << "Using previously generated project GUID:" << projectGUIDString;
         }
         // Create MSBuild project to compile and link all generated modules
         std::ofstream makefile((outputPath / "runner.vcxproj").str());

